### PR TITLE
Feat/BNGP-1658-upload-agent-authorisation

### PR DESCRIPTION
**Pull request by KiranVarma02**

### CODE REVIEW

**Comment by KiranVarma02** [comment]
duplicate - created new branch

File: packages/webapp/src/routes/land/landowner-permission-upload.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+} from '../../utils/helpers.js'
+
+const landownerPermissionUploadId = '#landownerPermissionUpload'
+
+function processSuccessfulUpload (result, request, h) {
+  let resultView = constants.views.INTERNAL_SERVER_ERROR
+  if (result[0].errorMessage === undefined) {
+    request.yar.set(constants.redisKeys.LANDOWNER_PERMISSION_UPLOAD_LOCATION, result[0].location)
+    request.yar.set(constants.redisKeys.LANDOWNER_PERMISSION_UPLOAD_FILE_SIZE, result.fileSize)
+    request.yar.set(constants.redisKeys.LANDOWNER_PERMISSION_UPLOAD_FILE_TYPE, result.fileType)
```
Comment (KiranVarma02) [inline packages/webapp/src/routes/land/landowner-permission-upload.js:18]: this

File: packages/webapp/src/__mock-data__/uploads/written-consent/wrong-extension copy.txt
Comment (AjinkyaJawalkar) [inline packages/webapp/src/__mock-data__/uploads/written-consent/wrong-extension copy.txt:1]: Remove this duplicate file

File: packages/webapp/src/routes/__tests__/land/landowner-permission-check.spec.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import * as azureStorage from '../../../utils/azure-storage.js'
+const url = constants.routes.LANDOWNER_PERMISSION_CHECK
+jest.mock('../../../utils/azure-storage.js')
+
+describe(url, () => {
+  describe('GET', () => {
+    it(`should render the ${url.substring(1)} view`, async () => {
+      await submitGetRequest({ url })
+    })
+    // it('should redirect to Start page if no data applicant data is available in session', async () => {
```
Comment (AjinkyaJawalkar) [inline packages/webapp/src/routes/__tests__/land/landowner-permission-check.spec.js:12]: Remove commented code

**Review by AjinkyaJawalkar — CHANGES_REQUESTED** [review]

**Comment by AjinkyaJawalkar** [comment]
Overall PR looks good to me apart from the minor code removal requested.

File: packages/webapp/src/routes/__tests__/land/landowner-permission-check.spec.js (hunk 2 above)
Comment (KiranVarma02) [reply packages/webapp/src/routes/__tests__/land/landowner-permission-check.spec.js:12]: done

**Review by AjinkyaJawalkar — APPROVED** [review]
PR looks good to me. As discussed earlier we need a approval on the same from lead developer

**Comment by KiranVarma02** [comment]
created new branch from master and cherry picked commits from this branch. I will create new PR from new branch

**Pull request closed without merging**